### DISCUSSION TITLE
CRITICAL_PROCESS_DIED BSOD

### DISCUSSION THREAD

**Discussion by papaj-na-wrotkach** [Atlas-OS/Atlas#1218] [Help & Support]
### Before continuing...

- [X] I agree to follow Atlas' [Code of Conduct](https://github.com/Atlas-OS/.github/blob/main/profile/CODE_OF_CONDUCT.md)
- [X] I have searched our [issue tracker](https://github.com/Atlas-OS/Atlas/issues?q=is%3Aissue) to verify that the issue hasn't been reported already
- [X] I have searched the [Atlas documentation](https://docs.atlasos.net) beforehand for a solution
- [X] I am on the latest version of Atlas, and didn't do an unsupported upgrade

### Description

I installed Windows 23H2, and followed all the steps from the installation guide.

### Steps to reproduce

I don't really know how to reproduce this problem. All I can say is that the BSOD usually happens when I am playing games, so it might be related to the GPU drivers. However, the system is only used for gaming, so I could be wrong.

I have had some other problems with my hard drives (on Linux) and they could also be the cause of the BSOD. I'm going to try playing games with the drives disconnected from now on to see if this is the case and report back here. If it is not then that's an issue with Atlas, Windows, or GPU drivers.

### Expected behavior

No issues when playing games

### Actual behavior

BSOD with code CRITICAL_PROCESS_DIED

### Atlas Edition

Atlas for Windows 11 23H2

### CPU
- [AMD Ryzen 5 1600 AF](https://www.techpowerup.com/cpu-specs/ryzen-5-1600af.c2652)
### RAM
- GOODRAM IRDM X 8GB DDR4 3200MHz CL16 DIMM
- GOODRAM IRDM X 16GB DDR4 3200MHz CL16 DIMM
### STORAGE
- [Silicon Power P34A80 256 GB](https://www.techpowerup.com/ssd-specs/silicon-power-p34a80-256-gb.d725)
- some other HDDs I disconnected
### GPU
- [XFX RX 460 Double Dissipation 4 GB](https://www.techpowerup.com/gpu-specs/xfx-rx-460-double-dissipation-4-gb.b3782)
- AMD Software Adrenalin Edition 24.3.1 (tried both with and without RadeonSlimmer)

**Comment by he3als** [comment]
Hi, this issue isn't related to Atlas. I'm going to convert this into a discussion so you can still get help on the forum though

**Reply by papaj-na-wrotkach** [reply]
If vanilla Windows works fine and it only happens with Atlas then it IS issue with Atlas.
Also I just got a BSOD, my HDDs were disconnected so it is not problem with my hardware.

**Reply by he3als** [reply]
I can't find any plausible change in Atlas that could cause your issue. It's a generic one that's likely related to other factors.
Additionally, I need to learn how to reproduce your problem; otherwise, I'm unsure how to troubleshoot.

**Reply by AudiEtoffe** [reply]
This is a hardware crash, I see those most often with ram that cant hit expo/xmd

**Reply by AudiEtoffe** [reply]
you might be seeing it more with atlas than stock due to disabling of MIV memory integrity validation

**Reply by papaj-na-wrotkach** [reply]
@he3als My apologies, you were correct. The issue began occurring after installing AtlasOS. It did not occur on my previous Windows 11 system, but I have confirmed that now it also occurs on a non-modified Windows 11. The installation of Atlas might or might not be the source of the issue. I believe that it is not.

@AudiEtoffe Should I do a test with memtest? Also, could you explain what "hitting expo/xmd" means?

PS
I had a break from Windows for some time, hence the delay.

**Reply by AudiEtoffe** [reply]
> @he3als My apologies, you were correct. The issue began occurring after installing AtlasOS. It did not occur on my previous Windows 11 system, but I have confirmed that now it also occurs on a non-modified Windows 11. The installation of Atlas might or might not be the source of the issue. I believe that it is not.
> 
> @AudiEtoffe Should I do a test with memtest? Also, could you explain what "hitting expo/xmd" means?
> 
> PS I had a break from Windows for some time, hence the delay.

memtest is a good start. Expo/XMD is the built in overclock for ram to get it running to the advertised speeds, you need to enable it in bios as by default your ram will run a lot slower (but more stable)
